README.md

```diff
@@ -285,6 +285,8 @@ cirros | Cirros
 fedora | Fedora (amd64)
 fedora.arm64 | Fedora (arm64)
 fedora.s390x | Fedora (s390x)
+linux | Linux (Generic) Guest
+linux-legacy | Linux Legacy (Generic) Guest
```

Comment:
```suggestion
linux-legacy | Linux (Legacy) Guest
```

Reply:
I've changed a bit the names and descriptions, could you please take a look?